tests/test_selector.py

```diff
@@ -37,6 +39,15 @@ def test_simple_selection(self):
         self.assertEqual([x.extract() for x in sel.xpath("concat(//input[@name='a']/@value, //input[@name='b']/@value)")],
                          [u'12'])
 
+    @mock.patch('scrapy.selector.unified.warnings')
+    def test_deprecated_root_argument(self, warnings):
+        root = etree.fromstring(u'<html/>')
+        sel = self.sscls(_root=root)
+        self.assertIs(root, sel.root)
+        warnings.warn.assert_called_once_with(
+            'Argument `_root` is deprecated, use `root` instead',
+            mock.ANY, stacklevel=2)
```

Comment:
Is it possible to test it without mocks, like described in [docs](https://docs.python.org/2/library/warnings.html#testing-warnings)?